sympy/printing/tests/test_latex.py

```diff
@@ -692,12 +692,12 @@ def test_latex_functions():
     assert latex(elliptic_pi(x, y)**2) == r"\Pi^{2}\left(x\middle| y\right)"
 
     assert latex(Ei(x)) == r'\operatorname{Ei}{\left(x \right)}'
-    assert latex(Ei(x)**2) == r'\operatorname{Ei}^{2}{\left(x \right)}'
+    assert latex(Ei(x)**2) == r'\left(\operatorname{Ei} {\left(x \right)}\right)^{2}'
     assert latex(expint(x, y)) == r'\operatorname{E}_{x}\left(y\right)'
     assert latex(expint(x, y)**2) == r'\operatorname{E}_{x}^{2}\left(y\right)'
-    assert latex(Shi(x)**2) == r'\operatorname{Shi}^{2}{\left(x \right)}'
-    assert latex(Si(x)**2) == r'\operatorname{Si}^{2}{\left(x \right)}'
-    assert latex(Ci(x)**2) == r'\operatorname{Ci}^{2}{\left(x \right)}'
+    assert latex(Shi(x)**2) == r'\left(\operatorname{Shi} {\left(x \right)}\right)^{2}'
+    assert latex(Si(x)**2) == r'\left(\operatorname{Si} {\left(x \right)}\right)^{2}'
+    assert latex(Ci(x)**2) == r'\left(\operatorname{Ci} {\left(x \right)}\right)^{2}'
```

Comment:
Why does these changes while `Chi` does not? I think that maybe only `Function` should be parenthesized as per design, not other defined SymPy functions

Reply:
@haru-44  pointed out that we should define a dictionary with some defined functions, not just the ones defined in `accepted_latex_functions`, to solve such issue.